service/controller/v3/arm_templates/main.json

```diff
@@ -60,32 +60,6 @@
     "kubernetesAPISecurePort": {
       "type": "int"
     },
-    "masterNodes": {
-      "type": "array"
-    },
-    "workerNodes": {
-      "type": "array"
-    },
-    "masterDisablePasswordAuth": {
-      "type": "bool",
-      "defaultValue": true
-    },
-    "workerDisablePasswordAuth": {
-      "type": "bool",
-      "defaultValue": true
-    },
-    "masterCloudConfigData": {
-      "type": "secureString",
-      "metadata": {
-        "description": "Base64-encoded cloud-config data."
-      }
-    },
-    "workerCloudConfigData": {
-      "type": "secureString",
-      "metadata": {
-        "description": "Base64-encoded cloud-config data."
-      }
-    },
```

Comment:
This here is cleanup and setup of the ARM templates because they are managed differently now.